backend/posts/serializers.py

```diff
@@ -10,3 +10,12 @@ class Meta:
         model = Post
         fields = ('author', 'title', 'text')
         read_only_fields = ('author',)
+
+class CommentSerializer(serializers.ModelSerializer):
+    author = CustomUserSerializer(read_only=True)
+    post = PostSerializer(read_only=True)
```

Comment:
it should be post = serializers.PrimaryKeyRelatedField(queryset=Post.objects.all())